terraform/everykidinapark.gov.tf

```diff
@@ -51,7 +51,16 @@ resource "aws_route53_record" "everykidinapark_gov_everykidinapark_gov_txt" {
   type = "TXT"
   ttl = 300
   records = [
-     "v=spf1 -all",
+     "v=spf1 -all"
+  ]
+}
+
+resource "aws_route53_record" "everykidinapark_gov_dmarc_everykidinapark_gov_txt" {
```

Comment:
I would suggest everykidinapark_gov__dmarc_everykidinapark_gov_txt to match the hostname